source/agora/crypto/Bech32.d

```diff
@@ -0,0 +1,486 @@
+/*******************************************************************************
```

Comment:
Could you move the public functions at the top of this module ?
I expect there's one to encode, one to decode, and they should be the first thing one see when opening this module.